kobolds/biology.md

```diff
@@ -75,24 +83,34 @@ tools and creams to ease the process and to engage the help of partners in
 scratching where it is hard to reach.
 
 
-## Reproduction
+## Physiology
```

Comment:
Carl said:
> I feel like the anatomy section is missing a description of their holistic physical nature. Can kobolds jump? How high? Climb? Swim? How long can they hold their breath? How many calories do they eat per day? How long can they go without protein? Do they cook their food? Do they ferment food? Do they eat salt? How long can they survive in freezing temperatures? How long can they survive in extreme heat? How dense are their bones? Can they see in the dark? How far can they see? How well can they smell?

On the topic of caloric intake, he mentioned the Japanese unit _koku_ as a touch stone for ideas to avoid using the unit _(kilo)calorie_:
> One koku is the amount of rice one peasant family requires to survive during one year. Koku was a common currency for taxation in Sengoku times